helper-charts/common-test/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: common-test
 description: Helper chart to test different use cases of the common library
-version: 0.1.0
+version: 0.1.1
```

Comment:
```suggestion
```
This chart is never published and only used within the unit-test framework. The version number is only there because it can't be `ct install`-ed otherwise. We don't need to bump this.